plugins/woocommerce-admin/client/core-profiler/actions/tracks.tsx

```diff
@@ -97,6 +98,20 @@ const recordTracksBusinessInfoCompleted = (
 	} );
 };
 
+const recordTracksPluginsInstallationRequest = (
+	_context: CoreProfilerStateMachineContext,
+	event: Extract<
+		PluginsInstallationRequestedEvent,
+		{ type: 'PLUGINS_INSTALLATION_REQUESTED' }
+	>
+) => {
+	recordEvent( 'coreprofiler_store_extensions_continue', {
+		shown: event.payload.pluginsShown || [],
+		selected: event.payload.pluginsSelected || [],
+		unselected: event.payload.pluginsUnselected || [],
```

Comment:
Even though it's possible to infer some values, we have both selected and unselected so that it's easy to query in analytics.